pkg/cmd/server/origin/authorizer.go

```diff
@@ -66,5 +65,5 @@ func newAuthorizer(
 		nodeAuthorizer,
 		scopeLimitedAuthorizer)
 
-	return authorizer, subjectLocator
+	return authorizer, kubeSubjectLocator
```

Comment:
This no longer should take or return the subject locator since it does nothing with it.